fe/src/main/java/org/apache/doris/clone/TabletChecker.java

```diff
@@ -356,71 +368,72 @@ private void removePriosIfNecessary() {
      * when being scheduled.
      */
     public void repairTable(AdminRepairTableStmt stmt) throws DdlException {
-        Catalog catalog = Catalog.getCurrentCatalog();
-        Database db = catalog.getDb(stmt.getDbName());
-        if (db == null) {
-            throw new DdlException("Database " + stmt.getDbName() + " does not exist");
-        }
+        RepairTabletInfo repairTabletInfo = getRepairTabletInfo(stmt.getDbName(), stmt.getTblName(), stmt.getPartitions());
+        addPrios(repairTabletInfo.dbId, repairTabletInfo.tblId, repairTabletInfo.partIds, stmt.getTimeoutS() * 1000);
```

Comment:
Why not just pass the entire `repairTabletInfo`? instead of each field of `repairTabletInfo`?
And `addPrios()` can be a private method.